voms-mapfile-default

```diff
@@ -54,3 +54,4 @@
 "/lhcb/Role=lcgadmin/Capability=NULL" lhcbadmin
 "/lhcb/Role=production/Capability=NULL" lhcbprod
 "/lhcb/*" lhcb
+"/clas12/*" clas12
```

Comment:
Please add a newline here and we'll get this merged

Reply:
ok done